### PR TITLE
Add mainnet seeds to README

### PR DESCRIPTION
**Pull request by Olshansk**
## Description

reviewpad:summary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on contributions and
 
 ## MainNet Seeds
 
-MainNet seeds are being maintained by [NodeFleet](https://nodefleet.org/) and more details can be found at [docs.pokt.network/node/seeds/](https://docs.pokt.network/node/seeds/).
+MainNet seeds are being maintained by [NodeFleet](https://nodefleet.org/) and more details can be found at [mainnet.md](./docs/guides/mainnet.md).
```
Comment (msmania) [inline README.md]: s/docs/doc/

**Review by StevenMartinez94 — DISMISSED** [review]

**Review by msmania — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on contributions and
 
 ## MainNet Seeds
 
-MainNet seeds are being maintained by [NodeFleet](https://nodefleet.org/) and more details can be found at [docs.pokt.network/node/seeds/](https://docs.pokt.network/node/seeds/).
+MainNet seeds are being maintained by [NodeFleet](https://nodefleet.org/) and more details can be found at [mainnet.md](./doc/guides/mainnet.md).
```
Comment (msmania) [inline README.md]: Can you use  the link to https://github.com/pokt-network/pocket-seeds instead of a new file?

Comment (msmania) [inline README.md]: I found a typo in "Accessing TestNet" above.  Can you include a fix for that too?
doc/guide/testnet.md --> doc/guides/testnet.md

**Review by msmania — CHANGES_REQUESTED** [review]

**Review by msmania — APPROVED** [review]
Thanks!

**Pull request merged**